cmd/mev-boost/main.go

```diff
@@ -41,6 +41,8 @@ var (
 	relayTimeoutMs = flag.Int("request-timeout", defaultRelayTimeoutMs, "timeout for requests to a relay [ms]")
 	relayCheck     = flag.Bool("relay-check", defaultRelayCheck, "check relay status on startup and on the status API call")
 
+	configPath = flag.String("config", "", "configuration for proposer preferences")
```

Comment:
```suggestion
	configPath = flag.String("config", "", "configuration file for proposer preferences")
```

Reply:
Updated in 3eaa39a9fe2dc59163cfdd0d5c496a606396f4f8, I could not apply your suggestion due to rebase.